lib/ansible/modules/system/launchd.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (c) 2016, Brian Coca <bcoca@ansible.com>
+# (c) 2106, Theo Crevon (https://github.com/tcr-ableton)
+# (c) 2016, Björn Albers <bjoernalbers@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+
+DOCUMENTATION = '''
+module: launchd
+author:
+    - 'Ansible Core Team'
+    - 'Theo Crevon (@tcr-ableton)'
+    - 'Björn Albers <bjoernalbers@gmail.com>'
+version_added: '2.3'
+short_description: Manage Mac OS X launchd services.
+description:
+    - Manage (start, stop and restart) Mac OS X launchd services via launchctl.
+options:
+    name:
+        required: true
+        description:
+            - Name of the service.
+        aliases: [ 'service', 'label' ]
+    state:
+        required: true
+        choices: [ 'started', 'loaded', 'stopped', 'unloaded', 'restarted', 'reloaded' ]
+        description:
+            - C(started), C(loaded), C(stopped) and C(unloaded) are idempotent
+              actions that will not run commands unless necessary.
+              Launchd does not support C(restarted) / C(reloaded) natively, so
+              these will both trigger a stop and start as needed.
+requirements:
+    - target host OS must be Mac OS X
+    - you have to run the module as root via sudo
```

Comment:
...Or being root and running ansible locally.  (bcoca suggests the wording: "root privileges are required"